app/controllers/public/users_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Public
+  class UsersController < ApplicationController
+    def check_username
+      @user = User.find_by(name: params[:name])
```

Comment:
Can we make here case insensitive check?
E.g., now if email 'user@mail.com' exists we receive false for 'User@mail.com'.
The same question for user name

Reply:
Done